src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,103 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    @include text-normal;
+
+    border-radius: $border-radius;
+    padding: 14px 18px 16px 15px;
+    width: 320px;
+
+    color: var(--grey-dark-2);
+    background-color: var(--white);
+}
+
+.info {
+    fill: var(--blue);
+}
+
+.warning {
+    fill: var(--orange);
+}
+
+.error {
+    fill: var(--red);
+}
+
+.success {
+    fill: var(--green);
+}
+
+.header {
+    display: flex;
+    position: relative;
+
+    svg {
```

Comment:
Bad `css` as was described [here](https://github.com/santiment/san-ui/pull/35#discussion_r260613592) and point to do it like this. All necessary properties can be specified in the `.closeIcon`